download_traffic_data.py

```
@@ -0,0 +1,234 @@
+                               f"{sensor.unit.value}\n")
+                        
+                        measurement_count += 1
+                        total_measurements += 1
+                        measurement_progress.update(1)
+                        
+                        if measurement_count % 10000 == 0:
+                            f.flush()
+                    
+                    measurement_progress.close()
+                    if measurement_count > 0:
+                        logger.info(f"{sensor.parameter.value}: {measurement_count:,} measurements")
```

Ruff (0.12.2)

Undefined name `logger`

(F821)